src/hypergraph.jl

```diff
@@ -72,8 +98,8 @@ Removes a vertex from a given hyperedge for a hypergraph `h` and a given vertex-
 """
```

Comment:
so maybe add a comment that trying to remove a vertex from a hyperedge when it is not present is not an error, earlier we wanted it to throw an error.